dluhc-component-library/src/components/siteSelectionForm/api/s3.tsx

```diff
@@ -0,0 +1,37 @@
+import AWS from "aws-sdk";

```

Comment:
Sorry I should have been more specific. I meant moving this to `src/api/aws/api.ts`